### PR TITLE
Unify spelling for some translatable strings

### PR DESCRIPTION
**Pull request by esbrandt**
This PR changes different spellings of some translatable strings to define key terminology for consistency in translations. We have a good bunch of new/changed strings anyway from 2.0 -> 2.1, so it is a good opportunity to clean things up a bit.

### CODE REVIEW

**Comment by daschuer** [comment]
Thank you. LGTM.

File: src/dlgprefcontrolsdlg.ui [hunk 1]
```diff
@@ -203,7 +203,7 @@
         <item row="2" column="0">
          <widget class="QLabel" name="labelKeylockMode">
           <property name="text">
-           <string>Key lock mode</string>
+           <string>Keylock mode</string>
```
Comment (Pegasus-RPG) [inline src/dlgprefcontrolsdlg.ui]: "Key lock" is technically two words.

Comment (sblaisot) [inline src/dlgprefcontrolsdlg.ui]: as well as "replay gain", "United States", ... despite they are all written in one word ;)

**Comment by esbrandt** [comment]
I do not feel strongly about the actual spelling. As said, this is a proposal to define key terminology. Once defined, i`d like to go trough website and manual sources, and fix them accordingly (if applicable).

**Comment by Pegasus-RPG** [comment]
Everything else looks good to me, so just make everything "key lock" (two separate words,) and we can merge this.

**Comment by esbrandt** [comment]
It`s been a while...

> just make everything "key lock" (two separate words,)

Imo ``Keylock``  has become a proper name.
It is currently the default for Mixxx, VDJ, Mixvibes, Serato, and Traktor , according to their respective feature description. There are inconsistencies in the spelling among the other programs manual/knowledge bases too though.

Also i can see potential issues with translations, since we have already ``keylock``established (and in our translation memory/glossary). Using the two words separately may lead to awkward translations, translators are not necessarily djs.

**Comment by rryan** [comment]
Thanks @esbrandt -- this LGTM. (+1 to ```keylock```).

**Pull request merged**